presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -59,7 +63,7 @@ Property Name                   Description
 ``kafka.buffer-size``           Kafka read buffer size
 ``kafka.table-description-dir`` Directory containing topic description files
 ``kafka.hide-internal-columns`` Controls whether internal columns are part of the table schema or not
-``kafka.messages-per-split``    Number of messages that will be processed by single Presto split
+``kafka.messages-per-split``    Number of messages that are processed by each Presto split, defaults to 100.000, smaller numbers result in higher parallelization of the query processing
```

Comment:
> 100.000

might be confusing in the US...

Reply:
what should I use then 100,000 ? or 100000 ?

Reply:
Updated to use same style as most other docs (100000)